envs.py

```diff
@@ -40,15 +42,16 @@ def embed_obs(self, obs):
     def compute_reward(self):
         r = 0.0
         if self.prev_obs is not None:
-            # This is necesarry because the character might be dying during multiple frames
+            # This is necessary because the character might be dying during multiple frames
             if not isDying(self.prev_obs.players[self.pid]) and \
                isDying(self.obs.players[self.pid]):
                 r -= 1.0
             
-        #     # We give a reward of -0.01 for every percent taken. The max() ensures that not reward is given when a character dies
-        #     r -= 0.01 * max(0, self.obs.players[self.pid].percent - self.prev_obs.players[self.pid].percent)
+            # We give a reward of -0.01 for every percent taken and +0.01 for damage dealt. 
+            # The max() ensures that not reward is given when a character dies
+            r += -0.01 * max(0, self.obs.players[self.pid].percent - self.prev_obs.players[self.pid].percent) + 0.01 * self.obs.players[opponent_pid].percent
```

Comment:
qq wouldn't you need a difference here to make it be incentivized by opponent losing percentages? np.abs(self.prev_obs.players[opponent_pid].percent - self.obs.players[opponent_pid].percent)?